testsuite/cluster-test/src/deployment.rs

```diff
@@ -64,19 +64,30 @@ impl DeploymentManager {
 
     pub fn update_all_services(&self) -> Result<()> {
         for instance in self.cluster.all_instances() {
+            self.update_service(instance.peer_name().to_string())?;
+            thread::sleep(Duration::from_millis(200));
+        }
+        Ok(())
+    }
+
+    fn update_service(&self, service: String) -> Result<()> {
+        for _ in 0..10 {
```

Comment:
You should be able to use `retry::retry` here similar to https://github.com/libra/libra/blob/master/testsuite/cluster-test/src/deployment.rs#L126